OpenRA.Mods.Common/Traits/Attack/AttackTurreted.cs

```diff
@@ -20,12 +20,14 @@ public class AttackTurretedInfo : AttackFollowInfo, Requires<TurretedInfo>
 		public override object Create(ActorInitializer init) { return new AttackTurreted(init.Self, this); }
 	}
 
-	public class AttackTurreted : AttackFollow
+	public class AttackTurreted : AttackFollow, INotifyCreated
 	{
 		protected Turreted[] turrets;
 
 		public AttackTurreted(Actor self, AttackTurretedInfo info)
-			: base(self, info)
+			: base(self, info) { }
+
+		public void Created(Actor self)
```

Comment:
Yes.  I'm mentioning it again here because if you are correct, then we have may have a serious regression in our actor creation code that will need to be investigated and fixed.